lib/livebook/hubs.ex

```diff
@@ -129,7 +129,7 @@ defmodule Livebook.Hubs do
 
   defp disconnect_hub(hub) do
     Task.Supervisor.start_child(Livebook.TaskSupervisor, fn ->
-      Process.sleep(30_000)
+      Process.sleep(10_000)
```

Comment:
We should have a comment as to why we sleep.

Btw `Connection` has backoff set to 5s, is it guaranteed that 10s is small enough, that's not related?

Reply:
The smallest sleep time is `5500`, is what I tested and it took time to delete and show the error message without repeating.

But, the reason, I'm not sure if it's LV related or is because we're running inside a task.

Reply:
Do you remember what was the reason for the sleep in #1675?

Reply:
I didn't find any conversation about that, I'm not sure if I did a call with José about that.

@josevalim Do you remember if we talked about adding a sleep time in the task to terminate the children?

Reply:
I think it was to allow the hub to process any event or message that may be on going. For example, what happens if a notebook is talking to the hub? Do we close the notebook? Do we change its hub? Even if we do that, we want to make sure the process does not suddenly disappear. Although 5s is probably more than enough.

Reply:
Wouldn't it be enough to let the process handle all pending messages and stop?

For open notebooks, further communication would result in an error, which makes sense. Closing automatically is also fine.

Reply:
Not really, the process only handles messages in its current inbox and only if trapping exits. Any message sent concurrently to the shutdown may still not be processed and crash.

Reply:
Right, I was asking about the semantics, we could send a stop message instead and let the process stop itself once it handles prior messages.

Reply:
What we could do is to send a message to the process and then the process sends a message to itself in 5 seconds, but we wouldn't escape from the timeout.